packs/aws/sensors/sqs_sensor.py

```diff
@@ -32,9 +35,17 @@ def __init__(self, sensor_service, config=None, poll_interval=5):
                                            poll_interval=poll_interval)
 
     def setup(self):
-        self.input_queue = self._get_config_entry(key='keys ', prefix='sqs_sensor')
+        queues = self._get_config_entry(key='input_queues', prefix='sqs_sensor')
+
+        # XXX: This is a hack as from datastore we can only receive a string while
+        # from config.yaml we can receive a list
+        if isinstance(queues, six.string_types):
+            self.input_queues = [x.strip() for x in queues.split(',')]
+        else:
```

Comment:
input_queues can be a YAML list instead of a comma separated string. Sure someone can specify an object or boolean or something crazy but I think this code is fine.